.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
```

Comment:
How are we determining the rules here? I am curious if we have a common eslint ruleset at Dropbox.